INFO 07-29 07:43:01 [__init__.py:235] Automatically detected platform cuda.
Traceback (most recent call last):
  File "/root/backdoor_detection/.venv/bin/vllm", line 4, in <module>
    from vllm.entrypoints.cli.main import main
  File "/root/backdoor_detection/.venv/lib/python3.10/site-packages/vllm/entrypoints/cli/__init__.py", line 3, in <module>
    from vllm.entrypoints.cli.benchmark.latency import BenchmarkLatencySubcommand
  File "/root/backdoor_detection/.venv/lib/python3.10/site-packages/vllm/entrypoints/cli/benchmark/latency.py", line 5, in <module>
    from vllm.benchmarks.latency import add_cli_args, main
  File "/root/backdoor_detection/.venv/lib/python3.10/site-packages/vllm/benchmarks/latency.py", line 16, in <module>
    from vllm import LLM, SamplingParams
  File "/root/backdoor_detection/.venv/lib/python3.10/site-packages/vllm/__init__.py", line 64, in __getattr__
    module = import_module(module_name, __package__)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/root/backdoor_detection/.venv/lib/python3.10/site-packages/vllm/entrypoints/llm.py", line 20, in <module>
    from vllm.config import (CompilationConfig, ModelDType, TokenizerMode,
  File "/root/backdoor_detection/.venv/lib/python3.10/site-packages/vllm/config.py", line 35, in <module>
    from vllm.model_executor.layers.quantization import QuantizationMethods
  File "/root/backdoor_detection/.venv/lib/python3.10/site-packages/vllm/model_executor/__init__.py", line 4, in <module>
    from vllm.model_executor.parameter import (BasevLLMParameter,
  File "/root/backdoor_detection/.venv/lib/python3.10/site-packages/vllm/model_executor/parameter.py", line 10, in <module>
    from vllm.distributed import get_tensor_model_parallel_rank
  File "/root/backdoor_detection/.venv/lib/python3.10/site-packages/vllm/distributed/__init__.py", line 4, in <module>
    from .communication_op import *
  File "/root/backdoor_detection/.venv/lib/python3.10/site-packages/vllm/distributed/communication_op.py", line 9, in <module>
    from .parallel_state import get_tp_group
  File "/root/backdoor_detection/.venv/lib/python3.10/site-packages/vllm/distributed/parallel_state.py", line 150, in <module>
    from vllm.platforms import current_platform
  File "/root/backdoor_detection/.venv/lib/python3.10/site-packages/vllm/platforms/__init__.py", line 267, in __getattr__
    _current_platform = resolve_obj_by_qualname(
  File "/root/backdoor_detection/.venv/lib/python3.10/site-packages/vllm/utils/__init__.py", line 2539, in resolve_obj_by_qualname
    module = importlib.import_module(module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/root/backdoor_detection/.venv/lib/python3.10/site-packages/vllm/platforms/cuda.py", line 18, in <module>
    import vllm._C  # noqa
ImportError: libcudart.so.12: cannot open shared object file: No such file or directory
